sympy/printing/mathml.py

```diff
@@ -653,8 +653,8 @@ def _print_MatrixBase(self, m):
             return table
         brac = self.dom.createElement('mfenced')
         if self._settings["mat_delim"] == "[":
-            brac.setAttribute('open', '[')
             brac.setAttribute('close', ']')
+            brac.setAttribute('open', '[')
```

Comment:
What is the difference in output as a result of this change?

I wonder if it would be better to change the tests rather than change the code...

Reply:
The tests would need to be compatible with earlier versions and 3.8. There doesn't seem to be an easy way to ensure correctness in both otherwise.

Reply:
Switching the order of attribute ensures that the tests run as intended on all versions without any further change.

Reply:
Okay that makes sense

Reply:
In the current versions the attributes are printed in alphabetical order, independent of in which order they are added. Hence close comes before open as in `<mfenced close=']' open='['>`. 

I guess that they are now printed in the order they are added (which makes sense), so this indeed seems like the way to get tests that works in all versions. (Although, clearly, one would like to have the open attribute first in the tag for estetical reasons...)

Reply:
We can add a comment that it can be reverted once the lowest Python version we support is 3.8.

Reply:
Where should I add the comment? Consider that these are spread across multiple functions.